genshin/client/components/hoyolab.py

```diff
@@ -29,12 +29,19 @@ async def search_users(
         )
         return [models.PartialHoyolabUser(**i["user"]) for i in data["list"]]
 
-    async def get_hoyolab_user(self, hoyolab_id: int, *, lang: typing.Optional[str] = None) -> models.FullHoyolabUser:
+    async def get_hoyolab_user(
+        self, hoyolab_id: typing.Optional[int] = None, *, lang: typing.Optional[str] = None
```

Comment:
missing trailing comma - should be `lang: typing.Optional[str] = None,` <-

Reply:
Fixed.